docs/getting-started/github-apps/migrating-tools-to-github-apps.rst

```diff
@@ -0,0 +1,158 @@
+
+.. code:: yaml
+
+   version: 1.2
+   tools:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test.json
+
+The information above was filled out using the following:
+
+- ``subclass`` is based on the descriptor language your tool is described in. In this case, the tool is described in CWL.
+- ``primaryDescriptorPath`` is from ``CWL Path`` or ``WDL Path``, depending on the ``subclass``. 
+- ``testParameterFiles`` is from ``CWL Test Parameter File Path`` or ``WDL Test Parameter File Path``, depending on the ``subclass``.
+
+.. note:: 
+    Existing Dockstore tools can be described in two languages, CWL and WDL, but GitHub App tools can only be described in one language. If your existing tool is described in two languages, you can either pick one language
+    or register two GitHub App tools, one for each language.
+
+During the original registration for your tool, you may have filled out the ``Tool Name`` field shown in the picture below.
+
+.. figure:: /assets/images/docs/tool-name-field.png
+   :alt: Tool name field
+   :width: 60 %
+
+This field is required when you want to register multiple tools from the same repo, but you may have filled it out for other reasons. To check if the tool
+you want to migrate has a tool name, select the tool and look at the title on top as shown in the picture below.
+
+The title consists of:
+``<image registry>/<organization name>/<repository name>/<optional tool name>:<version name>``
+
+If you see a tool name inserted, you must include the name field in your ``/.dockstore.yml``.
+
+.. code:: yaml
+
+   version: 1.2
+   tools:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test.json
+        name: optional-name
+
+If you have multiple tools registered on Dockstore that stem from the same GitHub repo, a single ``/.dockstore.yml`` can be used to convert them.
+Again, you need to check for the ``Tool Name`` field being set because it's needed for multi tool repositories to ensure that the tool paths on Dockstore are unique.
+
+Let's say we want to convert these two tools that come from this `repository <https://github.com/dockstore/dockstore-tool-bamstats>`__.
+
+.. image:: /assets/images/docs/github-apps-multiple-tools.png
+
+.. image:: /assets/images/docs/github-apps-multiple-tools-with-name.png
+
+
+Your ``/.dockstore.yml`` would look like the following:
+
+.. code:: yaml
+
+   version: 1.2
+   tools:
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore.cwl
+        testParameterFiles:
+            - /test.cwl.json
+      - subclass: CWL
+        primaryDescriptorPath: /Dockstore2.cwl
+        testParameterFiles:
+            - /test.cwl.json
+        name: optional-name
+
+Deleting Your Existing Tool
```

Comment:
Change to "Archiving or Deleting your Existing Tool" to nudge folks toward archiving 
Also because we should be working on https://github.com/dockstore/dockstore/issues/2104 as a part of U24 eventually

Reply:
What would archiving mean in Dockstore right now? Using the old tool page to link to the new one?

Reply:
Right now? 
Yeah, link and making an "out of service" type message in the description.